lib/bots/versionbot.ts

```diff
@@ -795,7 +842,7 @@ export class VersionBot extends ProcBot {
 			let description = 'Missing or forbidden tags in commits, see `repository.yml`';
```

Comment:
Should this be moved to a constant further up the file, along with the other ContextStatusMessages?
(unless there's some good reason for this being `let` rather than `const`?)